### ISSUE TITLE
Greeting while joining the group

### ISSUE THREAD

**Issue by Berezhnyk** [lvivdotnet/IgarBot#2]
I propose to add humanity to Igar. He will be perceived not only as a soulless bot but also as an assistant in the observance of spiritual values. 
The example of a greeting:
[Welcome|Hello], [user]! I am glad to congratulate you on the holiday [holiday]. Your new GIUID is the following: [GUID]. Have a good [day|evening|night]!

**Comment by xfreed** [comment]
Considering that there are many people in the group, will not his message turn into spam or is it planned to do it for individual correspondence?

**Comment by d1mnewz** [comment]
@xfreed As I would assume, this only applies to a case when someone new joins which does not happen that frequently.

**Comment by Berezhnyk** [comment]
Correct, it should be applied only when a user joins a group. Updated the issue name
